ui/app/AppLayouts/Wallet/adaptors/CollectiblesSelectionAdaptor.qml

```diff
@@ -0,0 +1,158 @@
+import QtQuick 2.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1
+
+import SortFilterProxyModel 0.2
+
+/**
+  Adaptor transforming input flat model of collectibles into grouped model,
+  grouped by communities and collections to form expected by components like
+  e.g. TokenSelector.
+
+  1. Ownership submodels filtered to have only entries for given `account`
+  2. Total balance calculated for remaining ownership entries in submodels
+  3. (1) and exposed to top level model (2)
+  4. Grouping value exposed depending if token comes from community or not
+     (community collectibles are grouped by communtiyId, other by collectionId)
+  5. Entries with zero balance filtered out
+  6. Items are sorted by communityId and collectionId in order to group them correctly
+  7. Grouping by groupingValue
+  8. For community groups, group once again by collectionId
+  9. Expose groupName and type according of it's community group or collection
+  10. Expose sub-sub-groups count as a balance for sub-groups
+**/
+
+QObject {
+    id: root
+
+    /** Account key used for filtering **/
+    property string account
```

Comment:
I changed to `accountKey` because I think that we should consistently distinguish unique key of data (for which content shouldn't matter, the only assumption is uniqueness)  and other properties, even though they are also unique. Even providing the same content for roles `key` and `address` in accounts model fully ok because provides logical separation.